lib/Crypt/SysRandom.pm

```diff
@@ -34,11 +34,14 @@ if (eval { require Crypt::SysRandom::XS }) {
 			$offset += $read;
 		}
 		return $result;
-	};
+	}
+	*random_bytes = \&random_bytes_urandom;
 } else {
 	die "No source of randomness found";
 }
 
+delete @Crypt::SysRandom::{qw(random_bytes_win32 random_bytes_urandom)};
```

Comment:
I think I'd lean towards writing that as
```
undef &random_bytes_win32;
undef &random_bytes_urandom;
```

Reply:
Except that made the pod coverage test unhappy…

Reply:
I don’t like the `delete` variant either because it requires repeating the package name so I had once written it that way somewhere else – but found that it didn’t do what I intended it to. I didn’t remember exactly what, though, so I didn’t feel confident to disagree. I am now guessing it was that it that way only removes the sub body but leaves the sub declared (as if there had been a `sub random_bytes_win32;` forward declaration)?